performance/src/main/java/org/hibernate/validator/performance/BenchmarkRunner.java

```diff
@@ -29,12 +26,12 @@
  */
 public final class BenchmarkRunner {
 
-	private static final Stream<Class<?>> DEFAULT_TEST_CLASSES = Stream.of(
-			SimpleValidation.class,
-			CascadedValidation.class,
-			StatisticalValidation.class,
-			MultiLevelContainerValidation.class
-	);
+	private static final Stream<? extends Class<?>> DEFAULT_TEST_CLASSES = Stream.of(
+			"org.hibernate.validator.performance.simple.SimpleValidation",
```

Comment:
For all the BV 1 compatible classes, you could use `<Class>.class.getName()` so that we limit the issue with renaming to the BV 2 classes. it's not particularly efficient but it's not that important as it's just a startup cost.